Spore ModAPI/Spore/App/Thumbnail_cImportExport.h

```diff
@@ -51,6 +51,13 @@ namespace App
 		bool SavePNG(Resource::ResourceObject* pResource, RenderWare::Raster* pImage, Resource::Database* database, 
 			bool forceReplace = false, bool disableSteganography = false);
 
+		/// Reads the PNG file from the given file path, and adds the creation found within to the Sporepedia if it hasn't been added yet. 
+		/// 
+		/// @param path The full path to the file being read.
+		/// @param key Resource key to the creation being read.
+		/// @returns 'true' on success, 'false' if something failed.
+		bool ReadPNG(const char16_t* path, ResourceKey& key);
```

Comment:
Can you rename it to `ImportPNG`? To me, `ReadPNG` gives more the feeling of reading a PNG file and just extracting its contents.